File: uri/1 - Basics/2028/Main.cpp
#include<iostream>

using namespace std;

int main()
{
    int i, k, sum,n, TC=0;

      while(cin>> n)
    {
        TC++;
        sum=0;
        if(n==0)
        {
            sum++;
            cout<<"Caso "<<TC<<": "<<sum<<" numero"<<endl;
            cout<<"0"<<endl;
        }
        else
        {
            for(i=0; i<=n; i++)
            {
                for(k=1; k<=i; k++)
                {
                    sum++;
                }
            }
            cout<<"Caso "<<TC<<": "<<sum+1<<" numeros"<<endl;
            cout<<"0 ";
            for(i=0; i<=n; i++)
            {
                for(k=1; k<=i; k++)
                {
                    cout<<i;
                    if(k<n) cout<< " ";
                }
            }
            cout<<endl;
        }
        cout<<endl;
    }
    return 0;
}

File: uri/1 - Basics/2028/Main.java
import java.util.*;
import java.io.*;
class Main{
    public static void main(String []args) throws Exception
    {
        int i, k, sum,n, TC=0;
        BufferedReader in = new BufferedReader(new InputStreamReader(System.in));
        PrintWriter out = new PrintWriter(System.out);
        String line;
        while( (line = in.readLine()) != null)
        {
            n = Integer.parseInt(line);
            TC++;
            sum=0;
            if(n==0)
            {
                sum++;
                out.printf("Caso %d: %d numero\n", TC,sum);
                out.printf("0\n");
            }
            else
            {
                for(i=0; i<=n; i++)
                {
                    for(k=1; k<=i; k++)
                    {
                        sum++;
                    }
                }
                out.printf("Caso %d: %d numeros\n", TC, sum+1);
                out.printf("0 ");
                for(i=0; i<=n; i++)
                {
                    for(k=1; k<=i; k++)
                    {
                        out.printf("%d", i);
                        if(k<n) out.printf(" ");
                    }
                }
                out.printf("\n");
            }
            out.printf("\n");
            out.flush();
        }
    }
}